ksonnet-util/kausal.libsonnet

```diff
@@ -391,5 +392,11 @@ k {
           podAntiAffinity.requiredDuringSchedulingIgnoredDuringExecutionType.withTopologyKey('kubernetes.io/hostname'),
         ]).spec,
       },
+
+    podPriority(p):
```

Comment:
Perhaps add a comment that this only sets the pod priority if `enable_pod_priorities` is set to `true`.